projects/packages/connection-ui/changelog/add-volta-pin-node-version

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
Seems not really "minor". It doesn't change the code at all, it's just configuring a dev tool.
```suggestion
Significance: patch
```
Same in your other changelog files.

Reply:
Updated.